go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/fogo-sh/gutenbuild
+
+go 1.16
+
+require (
+	github.com/BurntSushi/toml v0.3.1
+	github.com/bytecodealliance/wasmtime-go v0.28.0
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
+	github.com/rs/zerolog v1.23.0 // indirect
+	github.com/sirupsen/logrus v1.8.1
```

Comment:
I believe zerolog should be a direct dependency & logrus should be removed - can you run `go mod tidy` to update this file?